VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -497,6 +502,27 @@ export abstract class WaitingRoomBaseDirective {
         );
     }
 
+    private setTitle(roomTransfer: RoomTransfer): void {
+        const room: string = roomTransfer.to_room;
+        const transferId = roomTransfer.participant_id;
+        if (this.participant.id === transferId) {
+            let title;
+            if (room.includes('JudgeConsultationRoom')) {
```

Comment:
combine this with JudgeJOHConsultationRoom as they are setting the same title

Reply:
done

Reply:
done